products/gitlab.md

```diff
@@ -19,6 +19,13 @@ auto:
 # Support of R = releaseDate(R+1)
 # This is quite predictable since releases are monthly (usually 22nd of every month).
 releases:
+-   releaseCycle: "16.0"
+    releaseDate: 2023-05-18
```

Comment:
This date is not correct (should be 2023-05-22) but it will not be possible to fix it as it is the git tag date that is automatically managed.

Reply:
That's okay for this PR.